atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownTestSuite.java

```diff
@@ -35,6 +37,10 @@
         })
 public final class OneNodeDownTestSuite extends NodesDownTestSetup {
 
+    @ClassRule
+    public final RuleChain ruleChain = SchemaMutationLockReleasingRule.createChainedReleaseAndRetry(
+            createCassandraKvs(), CONFIG);
```

Comment:
This one should probably be a standard Rule, not a ClassRule - as written I think this releases the lock at the end of the tests rather than after each test.

Not too sure what `FlakeRetryingRule` does as a ClassRule as well - I don't think it actually retries stuff, but could be mistaken!

Reply:
I think we actually want a class rule, becasue this test is flaking in the BeforeClass part. My aim was to have this part be retried. Not sure if that's plausible though?